doc/whats_new/v1.0.rst

```diff
@@ -307,6 +307,10 @@ Changelog
   when the variance threshold is negative.
   :pr:`20207` by :user:`Tomohiro Endo <europeanplaice>`
 
+- |enhancement| :func:`feature_selection.RFE.fit` accepts addition estimator
+  parameters that are passed directly to the estimator's fit function.
+  :pr:`20380` by :user:`Iván Pulido <ijpulidos>` and :user:`Felipe Bidu <fbidu>`
```

Comment:
If you are reusing code from the first contributor, do not forget to add it as an author as well.

Reply:
Oh, good catch. Yes, I'm mostly reapplying the original code to the current state of the code. I'll update the author